$NetBSD$

Identified from message <mail:20070302224040.GA2038@pp> on crm114-general list.

--- mailreaver.crm	2007-03-02 17:10:03.000000000 -0600
+++ /usr/pkg/share/crm114/mailreaver.crm	2007-03-02 17:08:56.000000000 -0600
@@ -316,7 +316,7 @@
     alius    #  nope, not in the explicit headers.  Check for it as a 
     #           comment in the Message-Id: header.
     {
-	match [:in_text:] <nomultiline> (:: :reaver_cacheid:)  \
+	match [:in_text:] <nomultiline> (:: :cacheid:)  \
 		/Message-Id:.*\(sfid-([[:graph:]]+)\)/
 	isolate (:cacheid:) 
 	isolate (:long_cacheid:) /:*:text_cache:\/texts\/:*:cacheid:/
